communication/src/messages.cpp

```diff
@@ -132,15 +132,26 @@ size_t Messages::hello(uint8_t* buf, message_id_t message_id, uint8_t flags,
 	return len;
 }
 
-size_t Messages::update_done(uint8_t* buf, message_id_t message_id, bool confirmable)
+size_t Messages::update_done(uint8_t* buf, message_id_t message_id, uint8_t* result, size_t result_len, bool confirmable)
```

Comment:
Any reason not to make the `result`, `result_len`, `data`, and `data_len` const?